src/styles/themes/selectBox.ts

```diff
@@ -0,0 +1,71 @@
+import type { ThemeOption } from '@types'
+
+type SelectBoxKeys = keyof typeof selectBox
+type SelectBoxValues = typeof selectBox[SelectBoxKeys]
+
+export type SelectBox = ThemeOption<SelectBoxKeys, SelectBoxValues>
+
+export const selectBox = {
+  medium: {
+    default: `
+      div:nth-child(1){ //triggerWrapper
+        height: 40px;
+        padding: 12px 10px;
+        border-radius: 6px;
+        font-size: 14px; font-weight: 500; line-height: 20px;
```

Comment:
```suggestion
       ${theme.body02M};
```
테마 객체 활용하면 좋을 것 같심다!